tests/agents/test_microchain.py

```diff
@@ -28,3 +52,44 @@ def test_buy_no() -> None:
     market = get_omen_binary_markets()[0]
     buy_yes = BuyNo()
     print(buy_yes(market.question, 0.0001))
+
+
+def test_replicator_has_balance_lt_0() -> None:
+    balance = GetWalletBalance().__call__(REPLICATOR_ADDRESS)
```

Comment:
`__call__` in Python allows you to literally call that object:

```suggestion
    balance = GetWalletBalance()(REPLICATOR_ADDRESS)
```

Should work, usually there is no need to call these double underscored functions manually. Similarly in other places.